CHANGES.md

```diff
@@ -1,3 +1,18 @@
+## edge-19.4.1
+
+* CLI
+  * Introduced an `upgrade` command! This allows an existing Linkerd Control Plane to be reinstalled or reconfigured.
+  * The `--proxy-auto-inject` flag  from the `install` command has been completely revamped
+  * The `inject` command proxy options are now converted into config annotations. The annotations ensure that these configs are persisted in subsequent resource updates
+  * Always show open TCP connections in the CLI
```

Comment:
Wording - how about: 
```
The number of open TCP connections is now shown when running `linkerd stat`
```

Reply:
I agree wording should be changed. In going a little closer to my existing wording, I will probably go with `The `stat` command now always shows the number of open TCP connections`